src/shadowbox/server/run_action.sh

```diff
@@ -14,20 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo "** Currently broken, use yarn do shadowbox/docker/run instead **" >&2
-exit 1
-
 do_action shadowbox/server/build
 
 export LOG_LEVEL="${LOG_LEVEL:-debug}"
 export SB_PUBLIC_IP="${SB_PUBLIC_IP:-$(curl https://ipinfo.io/ip)}"
 # WARNING: The SB_API_PREFIX should be kept secret!
 export SB_API_PREFIX=TestApiPrefix
 export SB_METRICS_URL=https://metrics-test.uproxy.org
-export SB_STATE_DIR=/tmp/outline
+export SB_ROOT_DIR=${SB_ROOT_DIR:-$(mktemp -d)}
+mkdir -p ${SB_ROOT_DIR}/bin ${SB_ROOT_DIR}/persisted-state
+
+cp third_party/prometheus/prometheus ${SB_ROOT_DIR}/bin/prometheus
+SS_VERSION=1.0.8
```

Comment:
How are we going to keep this up to date and in sync with the image?
Maybe we need a download script that is shared.

Reply:
Since the Dockerfile needs SS_VERSION, the best way to do this was to make a tiny script which just echoes the version string to stdout.  Refactor done in 45ca40a